datastore/google/cloud/datastore_v1/gapic/datastore_client.py

```diff
@@ -151,8 +151,8 @@ def __init__(self,
     # Service calls
     def lookup(self,
                project_id,
-               keys,
                read_options=None,
+               keys=None,
```

Comment:
@chemelnucfin Is this change not in `master` yet? Wasn't that required for #4348?

Reply:
@chemelnucfin Why is this change here? The autogen was updated in #4348 so you should not be touching these files.